airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -184,21 +194,24 @@ def _upload_to_gcs(self, files_to_upload):
             hook.upload(self.bucket, object, tmp_file_handle.name, 'application/json')
 
     @classmethod
-    def convert_types(cls, value):
+    def convert_types(cls, value, is_binary=False):
         """
         Takes a value from MySQLdb, and converts it to a value that's safe for
         JSON/Google cloud storage/BigQuery. Dates are converted to UTC seconds.
-        Decimals are converted to floats.
+        Decimals are converted to floats. Binaries are converted to base64-encoded
+        strings.
         """
         if type(value) in (datetime, date):
             return time.mktime(value.timetuple())
         elif isinstance(value, Decimal):
             return float(value)
+        elif is_binary:
+            return base64.b64encode(value).decode()
```

Comment:
Curious about the decode() call on this

Reply:
In PY3, this decodes `bytes` to `str`, since b64encode takes an input of type `bytes` and also returns an encoded result of type `bytes` as well.